wheelhouse/README.txt

```diff
@@ -0,0 +1,43 @@
+Building Python Wheels
+======================
+
+For the highest compatibility, NEURON wheels are built in a
+manylinux1 image. Since the generic docker image is very basic
+(centos5) a new image, which brings updated libssl, cmake3 (3.12),
+ncurses, openmpi and mpich was prepared and made available at
+docker.io/pkumbhar/neuron_wheel
+
+Steps:
```

Comment:
I propose the addition of
```
0. Install Docker. On ubuntu...
sudo apt install docker.io
sudo groupadd docker
sudo usermod -aG docker $USER
#logout and log back in
```

Reply:
Addressed in 4d0dbd0c411114d6cf516c9e06b7dd5f5c6d5b9e